### PR TITLE
clean up third party links

### PR DESCRIPTION
**Pull request by christinaausley**
## Description

Tidying up third party links via monthly review.

## When should this change go live?

- [ ] This is a bug fix, security concern, or something that needs **urgent release support**.
- [ ] This is already available but undocumented and should be released within a week.
- [ ] This on a **specific schedule** and the assignee will coordinate a release with the DevEx team. (apply `hold` label or convert to draft PR)
- [ ] This is part of a scheduled alpha or minor. (apply alpha or minor label)
- [x] There is **no urgency** with this change and can be released at any time.

## PR Checklist

- [x] My changes are for an **already released minor** and are in `/versioned_docs` directory.
- [x] My changes are for the **next minor** and are in `/docs` directory (aka `/next/`).

- [ ] My changes require an [Engineering review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned the Engineering DRI or delegate.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @camunda/tech-writers as a reviewer.

### CODE REVIEW

File: docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ As a result, you need to include the SDK artifacts as follows in Maven:
 
 The SDK provides a [pre-packaged runtime environment](/components/connectors/custom-built-connectors/connector-sdk.md#pre-packaged-runtime-environment)
 that you can start manually. With version 0.3.0, this runtime moves from the [SDK repository](https://github.com/camunda/connector-sdk/tree/stable/0.2/runtime-job-worker)
-to [Spring Zeebe](https://github.com/camunda-community-hub/spring-zeebe/tree/master/connector-runtime). This also means that the provided runtime now is
+to [Spring Zeebe](https://github.com/camunda/connectors/blob/main/connector-runtime/README.md). This also means that the provided runtime now is
```
Comment (akeller) [inline docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md:77]: Oh goodness. I know the Connector runtime was still a part of the project in [8.2](https://github.com/camunda-community-hub/spring-zeebe/tree/8.2.0/connector-runtime) but it disappears in 8.3. I _think_ linking to 8.2 is more appropriate here, but you may want to open a backlog item to discuss cleaning up these update guides with @sbuettner and team.

I don't have a clear mapping of what Connector version goes to what C8 version to suggest it myself without digging.

Comment (sbuettner) [reply docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md:77]: @akeller As Connectors was only officially released with 8.3 and some of the links are outdated anyway (connectors-sdk was for exampled also moved into the mono repo) should we maybe clean up the docs and remove the links?

Comment (akeller) [reply docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md:77]: @sbuettner great idea. Can you sync with @christinaausley on this and open a separate PR?

For this PR, would my suggestion be ok? Or keep the change as-is?

Comment (sbuettner) [reply docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md:77]: Shouldnt the name be rather `Connector Runtime`?

Comment (christinaausley) [reply docs/components/connectors/custom-built-connectors/update-guide/020-to-030.md]: Adjusted in latest commit 👍 Let me know how I can assist with the clean-up.

**Review by akeller — APPROVED** [review]

**Review by sbuettner — APPROVED** [review]
Thanks!

**Pull request merged**